packages/ui-text-area/src/TextArea/index.tsx

```diff
@@ -318,31 +282,29 @@ class TextArea extends Component<TextAreaProps> {
         defaultValue={defaultValue}
         placeholder={placeholder}
         ref={(textarea, ...args) => {
-          // @ts-expect-error ts-migrate(2339) FIXME: Property '_textarea' does not exist on type 'TextA... Remove this comment to see the full error message
           this._textarea = textarea
-          // @ts-expect-error ts-migrate(2532) FIXME: Object is possibly 'undefined'.
-          textareaRef.apply(this, [textarea].concat(args))
+          if (typeof textareaRef === 'function') {
+            textareaRef.apply(this, [textarea, ...args])
+          }
         }}
-        // @ts-expect-error ts-migrate(2322) FIXME: Type '{ width: string | undefined; resize: "none" ... Remove this comment to see the full error message
         style={style}
         id={this.id}
         required={required}
         aria-required={required}
-        aria-invalid={this.invalid ? 'true' : null}
+        aria-invalid={this.invalid ? 'true' : 'false'}
```

Comment:
Hm, I would rather set this to `undefined`, because it might be an a11y change if we explicitly set it to `false`.

Reply:
It should not be a problem, but better safe than sorry